### PR TITLE
Nabla Shader Compile Tool [WIP]

### PR DESCRIPTION
**Pull request by AnastaZIuk**

### CODE REVIEW

File: tools/ndt/main.cpp [hunk 1]
```diff
@@ -1 +1,52 @@
...
+    {
+        if (args[i] == "-no-nbl-builtins")
+            return true;
+    }
+    return false;
+}
+
+
+
+int main(int argc, char* argv[])
```
Comment (devshgraphicsprogramming) [inline tools/ndt/main.cpp:26]: use ApplicationFramework

File: tools/ndt/main.cpp [hunk 2]
```diff
@@ -3,15 +3,17 @@
...
 #include <cstdlib>
-#include <vector>
+
 
 
 bool no_nbl_builtins;
 
 
-bool noNblBuiltinsEnabled(const std::vector<std::string>& args)
+bool noNblBuiltinsEnabled(const core::vector<std::string>& args)
```
Comment (devshgraphicsprogramming) [inline tools/ndt/main.cpp:16]: make it a method and you don't need to pass the `argv` around since its a member

Comment (nahiim) [reply tools/ndt/main.cpp:16]: OK. I think I'll just remove it and write the loop directly.

Comment (devshgraphicsprogramming) [reply tools/ndt/main.cpp]: anyway argument parsing is @Hazardu's job, you can hardcode all inputs and stuff meant to be gathered from cmdline

**Pull request closed without merging**